openassessment/locale/en/LC_MESSAGES/django.po

```diff
@@ -1,18 +1,18 @@
-# SOME DESCRIPTIVE TITLE.
-# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
-# This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# #-#-#-#-#  django.po (edx-ora2)  #-#-#-#-#
+# edX translation file
+# Copyright (C) 2018 edX
+# This file is distributed under the GNU AFFERO GENERAL PUBLIC LICENSE.
+# EdX Team <info@edx.org>, 2018.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
-"Report-Msgid-Bugs-To: \n"
+"Project-Id-Version: edx-ora2\n"
```

Comment:
This is set to `edx-platform` in other files. Do we know how that was set previously? Can we fix them to be consistent?

Reply:
I checked the some old commits and it seems it was set to edx-platform before. But as it is set to edx-platform in edx-platform repo should not we set it edx-ora2?.

Reply:
I'm not sure what it should be, as the gettext docs aren't clear (https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html). If edx-platform has been ok using 'edx-platform', then I have no reason to believe that using 'edx-ora2' wouldn't be ok here.